blueocean-bitbucket-pipeline/src/test/java/io/jenkins/blueocean/blueocean_bitbucket_pipeline/cloud/BbCloudWireMock.java

```diff
@@ -11,6 +11,8 @@
  */
 public class BbCloudWireMock extends BitbucketWireMockBase {
 
+    protected static String USER_KEY = "{1c5c9255-d59f-47e2-b5c7-52269c0332b9}"; // vivekp7;
```

Comment:
Maybe change these from `_KEY` to `_UUID`? Also probably nothing added by showing the old texts in comments, it might be confusing to future maintainers thinking it requires live real stuff set up by Vivek?

Reply:
yea fair enough, the comments were for me because i was starting to get confused by the uuid's everywhere. That's vivek's uuid so we didn't have to generate more fixtures.

Reply:
Right, but it's not actually going to the cloud right, that's the point of having the `wiremock`?

Reply:
Correct, i can delete the comment now that its done, i needed the comment to know which value to replace with which uuid at the time. The constants helped a lot. I'll rename and delete the comment.